app/client/cli/servicer.go

```diff
@@ -72,6 +72,8 @@ Will prompt the user for the *application* account passphrase`,
 			Aliases: []string{},
 			Args:    cobra.ExactArgs(4),
 			RunE: func(cmd *cobra.Command, args []string) error {
+				defer sessionCache.Stop()
```

Comment:
to avoid the linter error here, it might be better to just run this manually at the end
```go
return sessionCache.Stop()
```